xscen/aggregate.py

```diff
@@ -467,3 +491,282 @@ def spatial_mean(
         ds_agg.attrs["cat:processing_level"] = to_level
 
     return ds_agg
+
+
+@parse_config
+def produce_warming_level(
+    ds,
+    wl,
+    indicators,
+    window=20,
+    min_window=None,
+    tas_base_period=["1850", "1900"],
```

Comment:
A1: In defense of having defaults in general  (window and baseline), we want a Ouranos standard on how to calculate warming levels. It will make conversation easier if most of the time most people at Ouranos use the same window and baseline. To me, it make sense that our standard/ default shoud be the same as the IPCC Atlas.
A2: you are right.
A3: ok for baseline

Reply:
Comme discuté jeudi, j'ai laissé le défaut.